src/ida_pro_mcp/mcp-plugin.py

```diff
@@ -658,6 +658,44 @@ def list_globals(
     """List all globals in the database (paginated)"""
     return list_globals_filter(offset, count, "")
 
+class Import(TypedDict):
+    address: str
+    name: str
+    library: str
+
+@jsonrpc
+@idaread
+def list_imports(
+        offset: Annotated[int, "Offset to start listing from (start at 0)"],
+        count: Annotated[int, "Number of imports to list (100 is a good default, 0 means remainder)"],
+) -> Page[Import]:
+    """ List all imported symbols with their name and module (paginated) """
+    nimps = ida_nalt.get_import_module_qty()
+
+    rv = []
+    for i in range(nimps):
+        module_name = ida_nalt.get_import_module_name(i)
+        if not module_name:
+            module_name = "<unnamed>"
+
+        def imp_cb(ea, symbol_name, ordinal, acc):
+            if not symbol_name:
+                symbol_name = "<unnamed>"
+
+            acc += [{
+                "module": module_name,
+                "import": symbol_name,
+                "address": f"{ea:#x}",
+                "ordinal": f"#{ordinal}"

```

Comment:
Does the ordinal provide any kind of value? As far as I know this is just what IDA uses as an index to identify that specific import.